client/my-sites/site-monitoring/components/site-logs-toolbar/index.tsx

```diff
@@ -103,14 +103,15 @@ export const SiteLogsToolbar = ( {
 
 	const requestStatuses = [
 		{ value: '', label: translate( 'All' ) },
-		{ value: '200', label: translate( '200' ) },
```

Comment:
I'd appreciate a sanity check here but I can't see why these numbers should be translated - any translator would just use the default number: 

<img width="486" alt="Screenshot 2024-05-19 at 14 27 52" src="https://github.com/Automattic/wp-calypso/assets/43215253/bfb7b6d0-a729-4a33-9012-1d902dd288ea">

If the aim is to localise it based on decimal places, it might be better to use `formatNumber` - but that seems unnecessary for HTTP requests where you'd probably want the label to be exactly as shown instead.

Reply:
Some languages don't use Arabic digits and use different characters instead. I'm unsure if they translate HTTP codes, though. If it doesn't do harm, I would leave it as before.

Reply:
Ah, I see, thanks! I've switched it to `formatNumber` which should handle this without translators for each language needing to handle 9 different strings - do you think that might be okay? It seems to be the common approach for handling digits in Calypso: https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso+numberformat&type=code&p=1

Reply:
Sure, I've switched it back for now - I think that it localises it too, but the more I look into it, I'm not sure that this needs a `translate` at all (it looks like all Calypso languages use Arabic digits?). But I might be missing something, and possibly not in-scope for this PR!

Reply:
@Aurorum thanks for changing it. I've checked this more with the translations team and got feedback saying that HTTP error codes should be displayed as digits in all languages. Thus it's okay in the current PR, and we can optionally remove `translate()` call for them under a separate PR.

Reply:
Thanks for the review, and for asking the i18n team about this! I've proposed removing them in #90920. :)